lib/runtime/helpers_test.go

```diff
@@ -23,7 +27,7 @@ func TestValidateTransaction_IncludeData(t *testing.T) {
 	require.NoError(t, err)
 
 	validity, err := rt.ValidateTransaction(tx)
-	require.Nil(t, err)
+	require.NoError(t, err)
```

Comment:
I like this, I've been using require.Nil, I didn't even know about require.NoError, makes more sense.